src/ApiCallable.php

```diff
@@ -39,22 +39,22 @@
  */
 class ApiCallable
 {
-    const GRPC_CALLABLE_PARAM_COUNT = 3;
-    const GRPC_CALLABLE_METADATA_INDEX = 1;
-    const GRPC_CALLABLE_OPTION_INDEX = 2;
-    const GRPC_RESPONSE_STATUS_INDEX = 1;
+
+    const TRANSPORT_METHOD_PARAM_COUNT = 2;
+    const TRANSPORT_METHOD_OPTIONS_INDEX = 1;
 
     private static function setTimeout($apiCall, $timeoutMillis)
     {
         $inner = function () use ($apiCall, $timeoutMillis) {
             $params = func_get_args();
-            if (count($params) != self::GRPC_CALLABLE_PARAM_COUNT ||
-                !is_array($params[self::GRPC_CALLABLE_OPTION_INDEX])) {
+            if (count($params) != self::TRANSPORT_METHOD_PARAM_COUNT ||
+                !is_array($params[self::TRANSPORT_METHOD_OPTIONS_INDEX])
+            ) {
                 throw new InvalidArgumentException('Options argument is not found.');
             } else {
-                $timeoutMicros = $timeoutMillis * 1000;
-                $params[self::GRPC_CALLABLE_OPTION_INDEX]['timeout'] = $timeoutMicros;
-            }
+                    $timeoutMicros = $timeoutMillis * 1000;
+                    $params[self::TRANSPORT_METHOD_OPTIONS_INDEX]['timeout'] = $timeoutMicros;
```

Comment:
I got this slightly wrong, this should be 'timeoutMillis', because it is going to be passed to https://github.com/googleapis/gax-php/pull/104/files#diff-b729a08173cf725b2e82dbee0d1f03e5R67

The conversion to timeoutMicros happens here: https://github.com/googleapis/gax-php/pull/104/files#diff-7f453da75824436820c26d837b7f1303R160

Reply:
So it should just be this, correct?

```php
$params[self::TRANSPORT_METHOD_OPTIONS_INDEX]['timeout'] = $timeoutMillis;
```